config/jobs/periodic/housekeeping/cleanup-k8s-periodics.yaml

```diff
@@ -23,4 +23,5 @@ periodics:
               chmod +x /usr/local/bin/pvsadm
 
               # delete all the vms created before 4hrs
-              pvsadm purge vms --instance-id e13a10d2-9c2b-4d7b-b2fc-7a2b9edf1abf --dry-run --before 4h --ignore-errors --no-prompt
+              pvsadm purge vms --instance-id e13a10d2-9c2b-4d7b-b2fc-7a2b9edf1abf --before 4h --ignore-errors --no-prompt
+              pvsadm purge networks --instance-id e13a10d2-9c2b-4d7b-b2fc-7a2b9edf1abf --before 4h --ignore-errors --no-prompt
```

Comment:
Let's only purge the instances and not the network, though purge command takes before arguments but internally it does not really compare with the creation time, in fact networks does not have the creation time..

Reply:
I have removed the purge of networks and re-submitted it.